.gitignore

```diff
@@ -1 +1 @@
-.vscode/
\ No newline at end of file
+.vscode/
```

Comment:
Add `.DS_Store` to the .gitignore file and remove the binary from the repo.

Reply:
Done!